lib/plugins/inspec-habitat/Rakefile

```diff
@@ -0,0 +1,34 @@
+require 'rake/testtask'
```

Comment:
Make sure these tasks are ran in InSpec core, but leave integration test for ease of use.

Reply:
I elected to just remove all together and document the kitchen command.